src/main/java/com/csforge/sstable/Query.java

```diff
@@ -52,8 +55,12 @@
     public final CFMetaData cfm;
     public final Selection selection;
     public final ColumnFilter queriedColumns;
+    public AggregationSpecification aggregationSpec;
     public final Collection<File> path;
 
+    private static final SSTableReadsListener emptyReadListener = new SSTableReadsListener() {
```

Comment:
This is a required parameter for getScanner, although we ultimately don't make use of it.